service/gcs/runtime/runc/runc.go

```diff
@@ -324,13 +325,20 @@ func (c *container) GetAllProcesses() ([]runtime.ContainerProcessState, error) {
 	if err != nil {
 		return nil, err
 	}
+	l := logrus.WithField("cid", c.id)
```

Comment:
Nit: Why not just `WithFields`? Might be missing something

Reply:
Realized I only selected one line, I'm referring to how you just end up adding another field right below and logging